src/Gui/DlgExpressionInput.cpp

```diff
@@ -311,7 +309,7 @@ void DlgExpressionInput::checkExpression(const QString& text)
 
                 numberRange.throwIfOutOfRange(value);
 
-                ui->msg->setText(msg);
+                ui->msg->setText(Base::Tools::fromStdString(msg));
```

Comment:
Here you can use `QString::fromStdString` instead.

Reply:
Fixed on other similar places, too. Btw, should be remove `Base::Tools::fromStdString` in favour of `QString::fromStdString`? The same for `toStdString()`.